en/docs/administer/blue-green-data-encryption-keyrotation.md

```diff
@@ -0,0 +1,252 @@
+    |-----------------------------------|-------------------------------------------------------------------|--------------|
+    | Event publishers                  | `/repository/deployment/server/eventpublishers` files             | Password     | 
+    | Super tenant secondary userstores | `/repository/deployment/server/userstores/<userstore>` files      | Password     | 
+    | Tenant secondary userstores       | `/repository/tenants/<tenant_id>/userstores/<userstore>` files    | Password     |      
+    
+## Tables that support syncing
+
+At the moment below tables are supported to be synced during key rotation from the old setup to the new setup. Any other data won’t be persisted in the new setup.
+
+??? info "Tables that support syncing"
+
+    | **Table**                     | **Purpose**                                             | **Recommendation**                                                                                                                                                       |
+    |-------------------------------|---------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+    | IDN_IDENTITY_USER_DATA        | Identity claims when the identity data store is enabled | Usually recommended to sync if identity management features are enabled in the system.                                                                                   |
+    | IDN_OAUTH2_ACCESS_TOKEN       | OAuth 2.0 tokens                                        | Need to sync if the tokens created during the key rotation period needs to be valid after key rotation.                                                                         |
+    | IDN_OAUTH2_ACCESS_TOKEN_SCOPE | OAuth 2.0 scopes                                        | If the IDN_OAUTH2_ACCESS_TOKEN is synced, this table also needs to be synced.                                                                                            |
+    | IDN_OAUTH2_AUTHORIZATION_CODE | OAuth 2.0 authorization codes                           | Need to sync if the authorization codes created during the key rotation period need to be valid after key rotation. |
+
+
+!!! note
+    In this section, `<OLD_IS_HOME>` is the directory where the current Identity Server resides in, and `<NEW_IS_HOME>` is the directory where the copy of the current Identity Server resides in. `<KEY_ROTATION_REPO>` refers to the location [here](https://github.com/wso2/identity-tools/components/org.wso2.carbon.identity.keyrotation) and the `<KEY_ROTATION_TOOL>` refers to the location of the external tool
+
+## How To Create The Key Rotation Tool
+
+1.  Clone the repository [identity-tools](https://github.com/wso2/identity-tools)
+
+2.  Built it using maven by running the command `mvn clean install`
```

Comment:
```suggestion
2.  Build it using maven by running the command `mvn clean install`
```

Reply:
Addressed with https://github.com/wso2/docs-is/pull/2555/commits/e4826286f4b66bb74b49890f19dce24148b95808